test/fail_compilation/retscope.d

```diff
@@ -349,3 +349,71 @@ int* bar11(scope int** x) @safe
 }
 
 int* foo11(int* x) @safe { return x; }
+
+/******************************************/
+
+
+
```

Comment:
It's my new trick to avoid line numbers getting messed up when rebasing with the other PRs which add to the same file.